src/core/configProvider/defaultConfig.tsx

```diff
@@ -11,6 +11,12 @@ export const defaultConfig: Config = {
   // get internal URL origins from environment variables in the client app
   internalHrefOrigins: [],
   currentLanguageCode: LanguageCodeEnum.Fi,
+  fallbackBackgroundImageUrls: [
```

Comment:
could we somehow make not an array, but object with required properties, so first of all we force to add those images to the config in the actual projects, which are using the library. And on the other hand, if its named properties, its easier to use the specific images in specific components, for example one for Card, one for Sidebar and so on. Otherwise the array can be endless and how to define which image should be used where.

Reply:
That's a good idea and we can discuss about it. I'm a bit worried that it gets quite complicated with the types. Either we could allow it to be a Record type.`fallbackBackgroundImageUrls: Record<string, string[]>`, so you can determine any type of key string like 'card' and associate it with specific image url(s). But then you have to also add the type to the component: `<BackgroundImage type="card" url="...">` what heppens if type string 'card' is not found from the config record type? Should we throw error then?
Another option is to use predefined types:
```
fallbackBackgroundImageUrls: {
   card?: string[],
   sidebar?: string[]
}
```
But then we have update the cms config every time we want to add a new type.

Currently you can override the background image by the prop:
```
    <BackgroundImage
      url="somethin.jpg"
      customFallbackUrl="custom_fallback_image.jpg"
    />,
```  
I think that's the easiest way to override the fallback image url. But if it seems that for example sidebar really needs different image than the default one, then we can think about more customized configuration. What do you think? :)

Reply:
what if we just use one prop and say url={val1 || val2} ? In config we can make props mandatory so its required to set something, Niko suggested it. Otherwise we cannot control it anyhow you are right.